libs/langchain/langchain/llms/cerebriumai.py

```diff
@@ -91,7 +92,8 @@ def _call(
         **kwargs: Any,
     ) -> str:
         headers: Dict = {
-            "Authorization": self.cerebriumai_api_key,
+            "Authorization": self.cerebriumai_api_key
```

Comment:
hm i dont think this is right, right? this expression will evaluate to a boolean?